README.md

```diff
@@ -1,3 +1,68 @@
-# SE
-Презентация нашего проекта:
+# ML logs analyzer [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome#readme)
+[![License](https://img.shields.io/github/license/lizazhemchuzhina/SE.svg)]()
+<a href="https://github.com/lizazhemchuzhina/SE/graphs/contributors" target="_blank">
+<img src="https://img.shields.io/github/contributors-anon/lizazhemchuzhina/SE.svg" alt="GitHub contributors">
+</a>
+![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)
+## Introduction
+ML logs analyzer can be used for real-time monitoring of your project's logs in order to detect any changes and prevent
+failures in the future.
+
+## Background
+### Why?
+We want to know if a project is about to break down before it actually happens.
+### How?
+Monitor logs, analyze them, notice any significant changes.
+### What can you do with ML logs analyzer?
+The project is currently under development, so there are certain limitations to its functionality at the moment (but please keep in touch for future updates).
+1. Logs can be analyzed so that the most critical ones for failure prediction are selected.
+2. Users are registered in the system and can be added to working groups that correspond to projects.
+3. Logs corresponding to projects can be stored and queried.
+### Project presentation
 https://docs.google.com/presentation/d/1QaCWYwD5QQYgBTRzfzJL7RVKDnLpcdrF-FIhumKr3gk/edit?usp=sharing
+
+## Project ROADMAP
+https://venngage.net/ps/dym6Lzk3GPQ/ml-logs-analyzer
+
+## Installation
+Currently ML logs analyzer is available as a separate library. So you can simply clone and use it:
+
+```bash
+git clone https://github.com/lizazhemchuzhina/SE.git
```

Comment:
That is not the way to use library. What should I do once I clone the repository?

Reply:
It is intended to be used as a collection of source files.
I've added some examples so that it would be easier for users to figure out which features they might be interested in using.